requirements.txt

```diff
@@ -1,2 +1,4 @@
+-e git+https://github.com/datasciencebr/serenata-toolbox.git@6126436#egg=serenata-toolbox
```

Comment:
Yep. `-e` means editable mode, so it copy the package source code to your project. We don't need that here ; )